DnnWebsiteManagement/DnnWebsiteManagement.psm1

```diff
@@ -599,8 +604,30 @@ function New-DNNSite {
 
     if ($Domain -ne '') {
       if ($PSCmdlet.ShouldProcess($Name, 'Update Portal Aliases')) {
-        invokeSql -Query:"UPDATE $(getDnnDatabaseObjectName -objectName:'PortalAlias' -DatabaseOwner:$DatabaseOwner -ObjectQualifier:$ObjectQualifier) SET HTTPAlias = REPLACE(HTTPAlias, '$Domain', '$Name')" -Database:$Name
-        invokeSql -Query:"UPDATE $(getDnnDatabaseObjectName -objectName:'PortalSettings' -DatabaseOwner:$DatabaseOwner -ObjectQualifier:$ObjectQualifier) SET SettingValue = REPLACE(SettingValue, '$Domain', '$Name') WHERE SettingName = 'DefaultPortalAlias'" -Database:$Name
+        $aliasCounter = invokeSql -Query:"SELECT count(*) AliasCount FROM $(getDnnDatabaseObjectName -objectName:'PortalAlias' -DatabaseOwner:$DatabaseOwner -ObjectQualifier:$ObjectQualifier)" -Database:$Name
+        $aliasCount = $aliasCounter.aliasCount
+        $ProcessManual = ''
+
+        if ($Interactive.IsPresent) {
+          $ProcessManual = Read-Host -Prompt "Would you like to manually rename all $aliasCount portal aliases? (y/n)"
+          while("y","n" -notcontains $ProcessManual ) {
+            $ProcessManual = Read-Host -Prompt "Would you like to manually rename all $aliasCount portal aliases? (y/n)"
+          }
+        }
+        
+        if ($ProcessManual -eq 'y') {

```

Comment:
it looks to me like if I answer `'n'`, the automatic renaming is skipped

Reply:
The automatic renaming is handled in the else on line 628

Reply:
ah, I was mismatching the braces, thanks for setting me straight